alibuild_helpers/build.py

```diff
@@ -978,7 +978,7 @@ def downloadTask(p):
         additionalEnv=" ".join(
           "-e {}={}".format(var, quote(value)) for var, value in buildEnvironment),
         develVolumes=" ".join(
-          '-v "$PWD/$(readlink {pkg} || echo {pkg})":/{pkg}:ro'.format(pkg=quote(pkg))
+          '-v "$PWD/$(readlink {pkg} || echo {pkg})":/{pkg}:rw'.format(pkg=quote(pkg))
```

Comment:
I guess you meant the opposite, in the description?

Reply:
Yes, I meant read-write. Oops.